src/dawn/engine_dawn.cc

```diff
@@ -443,5 +430,28 @@ Result EngineDawn::CreateFramebufferIfNeeded() {
   return {};
 }
 
+Result EngineDawn::GetFrameBufferInfo(ResourceInfo* info) {
+  assert(info);
+
+  if (!render_pipeline_info_.fb_is_mapped)
+    return Result("Dawn: FrameBuffer is not mapped");
+
+  // TODO(dneto): Need to pass back a row_stride.
+  info->image_info.texel_stride = render_pipeline_info_.fb_texel_stride;
+  info->image_info.width = kFramebufferWidth;
+  info->image_info.height = kFramebufferHeight;
+  info->image_info.depth = 1U;
+  info->image_info.size_in_bytes = render_pipeline_info_.fb_texel_stride *
```

Comment:
This member doesn't exist in ResourceInfo.  I think you didn't compile this code.  :-)